evaluator/src/main/resources/osadl/osadl.rules.kts

```diff
@@ -70,7 +56,7 @@ fun PackageRule.LicenseRule.isOsadlMatrixCompliant(inboundLicenses: Collection<S
 
         override fun matches(): Boolean {
             val outbound = license.toString()
-            return inboundLicenses.all { matrix[it to outbound] in Compatibility.COMPLIANT }
+            return inboundLicenses.all { matrix.getValue(it).getValue(outbound) in Compatibility.COMPLIANT }
```

Comment:
This will fail if one of the licenses is not contained in the OSADL matrix which is pretty likely to happen in a real life project with ScanCode results. Maybe it would be a better option to return false in this case, and optionally add another rule to check if all licenses are covered by the matrix.

Reply:
Or simply leave the rule as-is, but only create a warning instead of an error?

Reply:
Or yet better (maybe that's what you mean): Only raise a violation if a license is explicitly defined as non-compliant. Yeah, that seems to make sense, will change.